JAICore/jaicore-ml/src/test/java/jaicore/ml/scikitwrapper/ScikitLearn_Wrapper_Test.java

```diff
@@ -180,11 +112,10 @@ public void loadOwnClassifierFromFileWithNamespace() throws Exception {
 		String importStatement = ScikitLearnWrapper.createImportStatementFromImportFolder(importfolder, true);
```

Comment:
This test does not test anything but only fails on an exception (and has "testsrc" path).

What do we want to check here? A reasonable test _could_ be to check the number of loaded classifiers or that particular classifiers are in fact present.